labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -70,26 +64,22 @@ object ListManipulationExercise01 {
    * - ... whichever way you like 
    * 
    */
-  def sortList[T <% Ordered[T]](list: List[T]): List[T] = {
-    error("fix me")
-  }
+  def sortList[T <% Ordered[T]](list: List[T]): List[T] = list.sorted
 
   /**
    * Check whether a given element in a list exists, i.e. elementExists(List("a", "b", "c"), "b") = true
    * Again, easy to implement using built-in functionality, but also possible to implement in your own free-style way.
    */
-  def elementExists[T](l: List[T], e: T): Boolean = {
-    error("fix me")
-  }
+  def elementExists[T](l: List[T], e: T): Boolean = l.foldLeft(false)((x,i) => if (x)
+    {x} else {if (i == e) true else false}) 
+      
 
   /**
    * Get all odd elements in the list, i.e. oddElements(List(1,2,3,4,5)) = List(1,3,5)
    * As always, use either build-in functions (for instance the filter method), or roll your own way via a
    * pattern match or some other method.
    */
-  def oddElements(iList: List[Int]): List[Int] = {
-    error("fix me")
-  }
+  def oddElements(iList: List[Int]): List[Int] = iList.foldLeft(List[Int]())((x,i) => if (i%2 == 1) x:+i else x)
```

Comment:
Oh, I see, I had misinterpreted this question. Anyways, appending to the right is usually not great if it can be avoided, since these Lists are basically linked lists. For this particular use case, `filter` can select only certain elements out of a list like you want to happen here.